CMakeLists.txt

```diff
@@ -192,6 +192,19 @@ set(c_util_h_files
     ./inc/c_util/uuid_string.h
 )
 
+if(WIN32)
+    #add worker_thread only on Windows (until we fix it)

```

Comment:
>(until we fix it) [](http://example.com/codeflow?start=39&length=17)

Do we have a task to extract the Win32 APIs we use out to the PAL (or change the implementation to use something already in the PAL)?

Reply:
ID Work Item Type Title Iteration Path State Assigned To Original Estimate
14543482 Task Have worker_thread be OS agnostic One\Custom\Azure Messaging Store\Iteration 261 To Do